### PR TITLE
Documentation/markdown

### PR DESCRIPTION
**Pull request by EluciusFTW**
All documentation transferred to markdown, with many small improvements along the way.

### CODE REVIEW

File: docs/modules/json.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Json Module
+This module provides functionality from the [json widget](https://spectreconsole.net/widgets/json) of _Spectre.Console_.
+
+Json can be printed in the console using the the json function:"
```
Comment (DanielmTheDev) [inline docs/modules/json.md:4]: Too many "the"s (same is true in the original documentation command)

File: docs/modules/status.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Status Module
+
+### Example
+```fs
```
Comment (DanielmTheDev) [inline docs/modules/status.md:4]: Example seems to be missing

**Review by DanielmTheDev — COMMENTED** [review]
I think it's a good idea to simply link to these files in the readme :)

**Review by DanielmTheDev — APPROVED** [review]

File: docs/modules/status.md (hunk 2 above)
Comment (EluciusFTW) [reply docs/modules/status.md:4]: Good catch. added one!

File: docs/modules/json.md (hunk 1 above)
Comment (EluciusFTW) [reply docs/modules/json.md]: That's that's true.

**Pull request merged**